cli/pkg/local/app.go

```diff
@@ -57,12 +65,21 @@ type App struct {
 	ProjectPath string
 }
 
-func NewApp(ctx context.Context, ver version.Version, verbose bool, olapDriver, olapDSN, projectPath string) (*App, error) {
-	// Setup a friendly-looking colored logger
+func NewApp(ctx context.Context, ver version.Version, verbose bool, olapDriver, olapDSN, projectPath string, logFormat LogFormat) (*App, error) {
+	// Setup a friendly-looking colored/json logger
+	var encoder zapcore.Encoder
 	conf := zap.NewDevelopmentEncoderConfig()
 	conf.EncodeLevel = zapcore.CapitalColorLevelEncoder
+
+	switch logFormat {
+	case LogFormatJSON:
+		encoder = zapcore.NewJSONEncoder(conf)
+	case LogFormatColor:
+		encoder = zapcore.NewConsoleEncoder(conf)
+	}
+
 	logger := zap.New(zapcore.NewCore(
-		zapcore.NewConsoleEncoder(conf),
+		encoder,
 		zapcore.AddSync(colorable.NewColorableStdout()),
 		zapcore.DebugLevel,
 	))
```

Comment:
Let's use `zap.NewProduction` for the `json` format, and only do friendly colored logging for the `color` logger